app/src/main/java/com/gathering/android/event/eventdetail/EventDetailComposable.kt

```diff
@@ -0,0 +1,214 @@
+package com.gathering.android.event.eventdetail
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Card
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.rememberAsyncImagePainter
+import com.gathering.android.R
+
+
+@Preview(showBackground = true)
+@Composable
+fun EventDetailPreview() {
+    EventDetail(
+        eventName = "Ani",
+        eventHostEmail = "animan@gmail.com",
+        description = "party",
+        photoUrl = "",
+        address = "101 Erskine Ave",
+        date = "15 Nov 2023",
+        time = "15:30",
+        acceptType = AcceptType.No,
+        {},
```

Comment:
Please pu names of every parameter in this function:
`{}` is unknown
use
`onXButtonClicked = {}`